### ISSUE TITLE
[Egg Request]: Valheim BepInEx

### ISSUE THREAD

**Issue by marindumont** [gOOvER/own-pterodactyl-eggs#4]
### Does this expand an already existing service

Yes

### Link to a game or software

https://store.steampowered.com/app/892970/Valheim/

### Links for the download

https://valheim.thunderstore.io/package/denikson/BepInExPack_Valheim/

### Links for the install docs

https://valheim.thunderstore.io/package/denikson/BepInExPack_Valheim/

### Are you willing to pay for this egg (Donation/Gamekey)?

Yes

**Comment by gOOvER** [comment]
there is allready a valheim bepinex egg

**Comment by marindumont** [comment]
yes but not compatible with the last vestion of valheim+... Or maybe you have a special source?

**Comment by marindumont** [comment]
no, a dev continue the branch on this github: https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.9.17

**Comment by gOOvER** [comment]
ty for the Link. i'll check it and Update the egg

**Comment by marindumont** [comment]
great, keep me informed... maybe try to contact the dev ton know if he continue the project... 
Can you provide me the egg for last vesrsion of valheim BepInEx server? on my side, i need to generate a manual vanilla valheim serv and copy BepInEx unixserv by ftp....

**Comment by gOOvER** [comment]
i started a new project for eggs and Images, since i left ptero discord.

atm i rewrite all my eggs.

After Work, i Check both eggs

**Comment by marindumont** [comment]
thanks for that... the problem is also when you try to run bepinex with mods on win 11, the anti-cheat is starting and crash the game... maybe you have a solution for that?
